src/Package/Util/Header/Version.php

```diff
@@ -90,14 +90,14 @@ public function fileHasVersionMacro($fname)
     public function getVersionFromHeader()
     {
         $headers = glob($this->package->getSourceDir().DIRECTORY_SEPARATOR.'*.h');
-        $pat = ',define\s+'.$this->macroName.'\s+"(.*)",i';
+        $pat = ',define\s+'.$this->macroName.'\s+(".*"|.*\b),i';
```

Comment:
@stof Correct.
@duncan3dc Please, could you apply `preg_quote` on `$this->macroName` please? Also, please, apply my suggestion. Yes, we can have `beta` or other suffix in the version number.